server/character.lua

```diff
@@ -9,10 +9,7 @@ end
 
 ---@param source Source
 local function giveStarterItems(source)
-    while not exports.ox_inventory:GetInventory(source) do
```

Comment:
Don't we still need this loop? This awaits for the inventory to be loaded

Reply:
Good point.